packages/foreman/foreman-installer/foreman-installer.spec

```diff
@@ -1,17 +1,15 @@
-# set and uncomment all three to set alpha tag
-#global alphatag RC1
-#global dotalphatag .%{alphatag}
-#global dashalphatag -%{alphatag}
+%global prerelease .develop
```

Comment:
This isn't entirely going to work I think. I think we need to drop the dot here and explicitly include it in Release. Otherwise the download link will end up looking for `1.19.0-.RC1.tar.bz2` That's why we had a `dotalphatag` and a `dashalphatag`.